web_graph_improved/README.md

```diff
@@ -0,0 +1,7 @@
+Charts, Improved.
+=================
+
+Various improvements to the graph view:
+
+ * Bar: can plot multiple variables (called measures in the odoo client)
+   when there is only one row group
```

Comment:
various improvement*s*, and then you list one. Is it an omission or are you planning more improvements in the future?